### ISSUE TITLE
module names in Train

### ISSUE THREAD

**Issue by dnkennedy** [ReproNim/repronim.github.io#32]
we need to make sure our Training Module names match in all occurrences. (i.e. 'Reproducibility Basics' versus 'Computational Basics').

**Comment by jbpoline** [comment]
Agreed - I also prefer to avoid 0-1-2-3 because the order can move a little ...
